src/core/internal/convert.d

```diff
@@ -34,8 +34,9 @@ private ubyte[] ctfe_alloc()(size_t n)
 }
 
 @trusted pure nothrow @nogc
-const(ubyte)[] toUbyte(T)(const ref T val) if (is(Unqual!T == float) || is(Unqual!T == double) || is(Unqual!T == real) ||
-                                        is(Unqual!T == ifloat) || is(Unqual!T == idouble) || is(Unqual!T == ireal))
+const(ubyte)[] toUbyte(T)(return const ref T val)
```

Comment:
The only actual change here is the added return to the val parameter. The code was just so horribly formatted, I had to do something about it.